birdhouse/config/finch/.gitignore

```diff
@@ -1 +1,5 @@
 wps.cfg
+config/**/*
```

Comment:
From the opposite perspective, a pattern like this prevents users from accidentally committing new config files.

I find that having a generic pattern that I re-use in all .dockerignore files is useful. And then I add specific exceptions only as needed. 

If you feel strongly I'm happy to change this back. This way of writing .dockerignore files is my preference but I don't have to force my preferences on this project.

Reply:
From the operation point of view, any silent or random bugs are the worst kind of bugs.

So forgetting to commit a new file because of the ignore rule is akin of a silent bug for me.  Also any update to this file going forward will not be detected as well so you'll end up adding a new ignore exception for this new file anyways.  Otherwise, this ignored file will always be silently "ignored" (no pun intended). 

Whereas committing a file that should not be committed will be spotted via review so it is more visible.  You'll have to modify `.gitignore` to include it, but again in the other scenario, you'll have to also modify `.gitignore` to exclude from the ignore all list. Same effort but this route is safer, no silent bugs.

My experience taught me to be very cautious with wildcard ignore rule.  Same deal with regexes.  You do not want it to match more than what you need so you have to craft very precise regex, else happy debugging time.  I often just avoid using regexes when I can.

I do not think we will add or rename or remove config files that often.  So the advantage of using wildcard ignore list to save us from frequently editing the `.gitignore` file is not there.

The risks of potential errors outweigh the benefits in this case.

Please avoid using wildcard ignore, unless the regex is every precise.